### ISSUE TITLE
A11y_AzureDataStudioSQL_Projects_Add View_ScreenReader: Voice Over does not announces the line number in code editor on which keyboard focus is there.

### ISSUE THREAD

**Issue by RohitJadhav-TCS** [microsoft/azuredatastudio#14524]
### Actual:
Voice Over does not announces the line number in code editor on which keyboard focus is there when navigated on it with the help of tab key.

Same issue is observed throughout the application where code editor is present

### Expected:
Voice Over should announce it as line 1 and then the information present in line when navigated on it with the help of tab key.

**GitHubTags**: #A11yMAS;#A11yTCS;#A11ySev2;#SH_AzureDataStudioSQL-MAC-Feb2021;#SQL Azure Data Studio;#MAC;#FTP;#DesktopApp;#WCAG1.3.1;#VoiceOver;#Element:Combobox;#AINotInScope;

### Environment Details:
Application Name: Azure Data Studio
Version: 1.26.1
Mac OS Version: Big Sur Version 11.2.2
Screen reader: Voice Over

### Repro Steps:

1. Launch the application-Azure Database Project and install extension SQL Database Projects.
2. Navigate to SQL Database Project Home Screen with heading "Projects".
3. Tab to connections select your server and connect to it.
4. Tab to Projects tab an right click on any created project.
5. Tab till "Add View" and hit enter
6. Start Voice Over and tab code editor line 1 and check the result. 

### User Impact:
Screen reader user will not get the complete information like on which line the keyboard focus is there,

“[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Comment by kisantia** [comment]
same issue as the one opened for Windows https://github.com/microsoft/azuredatastudio/issues/14414

**Comment by alanrenmsft** [comment]
dupe of https://github.com/microsoft/monaco-editor/issues/410

**Comment by Swati1700** [comment]
Verified the bug in the environment
Application Name: Azure Data Studio
macOS: Ventura 13.2
Issue is fixed and does not reproduced. Please find the not repro attachment below.
[Screen Recording (4).webm](https://user-images.githubusercontent.com/86710888/220585761-20bf6e27-4965-45e5-8f53-c3548edddf5d.webm)
Github Tags:#Closed;

**Comment by 12-shweta** [comment]
Logged new bug: https://github.com/microsoft/azuredatastudio/issues/22711

#Closed;
